### PR TITLE
docs: Updates for Gaia v14 (aka v14.1.0)

### PR DESCRIPTION
**Pull request by mmulji-ic**
## Description

Part of: [#2786](https://github.com/cosmos/gaia/issues/2786)

Updates for the Gaia v14 release.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: docs/getting-started/system-requirements.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# System requirements
+
```
Comment (MSalopek) [inline docs/getting-started/system-requirements.md:3]: This is done by most if not all OSs automatically and the swap file is scaled according to needs.

Can you elaborate why this is needed?

**Review by MSalopek — APPROVED** [review]

File: docs/getting-started/system-requirements.md (hunk 1 above)
Comment (mmulji-ic) [reply docs/getting-started/system-requirements.md:3]: This is in all the previous migration updates. For this release I've split out the requirements into a separate doc as I want to reduce the material we repeat in different locations.

File: docs/roadmap/README.md [hunk 2]
```diff
@@ -7,90 +7,23 @@ parent:
 
 # Previous Releases
 
-Please see the table below for library versions and other dependencies.
```
Comment (mmulji-ic) [inline docs/roadmap/README.md:10]: Deleted as this content is in the release notes. Don't want to maintain copies of release notes in other docs.

File: docs/roadmap/README.md [hunk 3]
```diff
@@ -7,90 +7,23 @@ parent:
...
 | Upgrade #2          | 07/08/20    |  2902000 | `cosmoshub-3` | [v0.32.x](https://github.com/tendermint/tendermint/releases/tag/v0.32.14)         | [v0.37.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.37.15)     | [v2.0.x](https://github.com/cosmos/gaia/releases/tag/v2.0.14)                   | n/a                      |
-| Stargate            | 18/02/21    |  5200791 | `cosmoshub-4` | [v0.34.x](https://github.com/tendermint/tendermint/releases/tag/v0.34.3)          | [v0.40.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.1)      | [v4.0.x](https://github.com/cosmos/gaia/releases/tag/v4.0.6)                   | _Included in Cosmos SDK_ |
+| [Stargate](https://github.com/cosmos/gaia/releases/tag/v4.0.0)           | 18/02/21    |  5200791 | `cosmoshub-4` | [v0.34.x](https://github.com/tendermint/tendermint/releases/tag/v0.34.3)          | [v0.40.x](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.1)      | [v4.0.x](https://github.com/cosmos/gaia/releases/tag/v4.0.6)                   | _Included in Cosmos SDK_ |
```
Comment (mmulji-ic) [inline docs/roadmap/README.md:18]: Added links to the releases so that the release notes can be easily accessed.

**Review by sainoe — APPROVED** [review]

**Pull request merged**